src/components/Inputs/Select.tsx

```diff
@@ -386,23 +419,22 @@ export const OptionSelect = styled.div`
 `
 
 export const OptionItem = styled.button<{ selected: boolean; focused: boolean; invisible?: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   padding: var(--spacing-4);
   cursor: pointer;
-  color: ${({ selected }) => (selected ? 'var(--color-white)' : 'inherit')};
+  color: ${({ theme }) => theme.font.primary};
   user-select: none;
   text-align: left;
-  background-color: ${({ theme, selected, focused }) =>
-    selected ? theme.global.accent : focused ? theme.bg.accent : theme.bg.primary};
+  background-color: ${({ theme, focused }) =>
+    focused ? theme.bg.accent : colord(theme.bg.primary).alpha(0.4).toHex()};
   visibility: ${({ invisible }) => invisible && 'hidden'};
+  font-weight: ${({ theme, selected }) => selected && 'var(--fontWeight-semiBold)'};
 
   &:not(:last-child) {
     border-bottom: 1px solid ${({ theme }) => theme.border.primary};
   }
-
-  &:focus {
-    background-color: ${({ theme, selected }) =>
-      !selected ? theme.bg.accent : colord(theme.global.accent).lighten(0.05).toRgbString()};
-  }
```

Comment:
By removing this we cannot visibly see which item the user navigated to using the tabs

Reply:
Fixed in #614